dali/kernels/test/block_setup_test.cc

```diff
@@ -299,11 +299,18 @@ TEST(BlockSetup, SetupBlocks_Uniform_CDHW) {
   });
 
   BlockSetup<3, 0> setup;
-  ivec3 def_block_size = { 256, 256, 1 };
+  ivec3 def_block_size = { 256, 256, D };
   setup.SetDefaultBlockSize(def_block_size);
   static_assert(setup.tensor_ndim == 4, "Incorrectly inferred tensor_ndim");
   setup.SetupBlocks(TLS);
   ASSERT_TRUE(setup.IsUniformSize());
+
+  while (volume(def_block_size) > 0x40000) {
```

Comment:
`0x40000` should better be a named constant

Reply:
It's not a good unit test, because it's actually testing the implementation. In such cases, I'd even say I like magic constants in tests - at least everything is clearly visible, not hidden behind layers of meaningless names.  I removed the magic value from Setup though.